src/Authentication/Authenticators/Session.php

```diff
@@ -922,17 +922,17 @@ protected function rememberUser(User $user): void
             $user,
             $selector,
             $this->hashValidator($validator),
-            $expires
+            $expires->format('Y-m-d H:i:s')
         );
 
         $this->setRememberMeCookie($rawToken);
     }
 
-    private function calcExpires(): string
+    private function calcExpires(): Time
     {
         $timestamp = Time::now()->getTimestamp() + setting('Auth.sessionConfig')['rememberLength'];
 
-        return Time::createFromTimestamp($timestamp)->format('Y-m-d H:i:s');
+        return Time::createFromTimestamp($timestamp);
```

Comment:
@kenjis This issue is not related to this PR, but I realized it now.
It seems that method `createFromTimestamp()` is not affected by changing the value of `App::$appTimezone` .

For example, by changing the value to App::$appTimezone = 'Asia/Tehran', I would expect the `createFromTimestamp()` method to be affected, but **timezone** is still set to 'UCT', which doesn't seem right. look :
https://github.com/codeigniter4/CodeIgniter4/blob/7d50bd33056cd54aef7f1905c1304947409ef078/system/I18n/TimeTrait.php#L266-L272

To fix it, you must do the following:
```diff
+use Config/App;
-return Time::createFromTimestamp($timestamp);
+return Time::createFromTimestamp($timestamp, config(App::class)->appTimezone);
```
Or make changes in CI4 
https://github.com/codeigniter4/CodeIgniter4/blob/7d50bd33056cd54aef7f1905c1304947409ef078/system/I18n/TimeTrait.php#L266-L272

Reply:
I sent a PR: https://github.com/codeigniter4/CodeIgniter4/pull/8544

Reply:
Fixed by v4.4.6.